tests/core/data/test_parquet_data_node.py

```diff
@@ -300,6 +300,44 @@ def test_get_system_folder_modified_date_instead_of_last_edit_date(self, tmpdir_
 
         os.unlink(temp_file_path)
 
+    @pytest.mark.skipif(not util.find_spec("fastparquet"), reason="Append parquet requires fastparquet to be installed")
+    @pytest.mark.parametrize(
+        "content",
+        [
+            ([{"a": 11, "b": 22, "c": 33}, {"a": 44, "b": 55, "c": 66}]),
+            (pd.DataFrame([{"a": 11, "b": 22, "c": 33}, {"a": 44, "b": 55, "c": 66}])),
+        ],
+    )
+    def test_append_pandas(self, parquet_file_path, default_data_frame, content):
+        dn = ParquetDataNode("foo", Scope.SCENARIO, properties={"path": parquet_file_path})
+        assert_frame_equal(dn.read(), default_data_frame)
+
+        dn.append(content)
+        assert_frame_equal(
+            dn.read(),
+            pd.concat([default_data_frame, pd.DataFrame(content, columns=["a", "b", "c"])]).reset_index(drop=True),
+        )
```

Comment:
I was more thinking abt something like that, to be sure we are properly catching and logging the error. What do you think?

```suggestion
        if not util.find_spec("fastparquet"):
            with pytest.raises(FastParquetRequiredError):
                dn.append(content)
        else:
            dn.append(content)
            assert_frame_equal(
                dn.read(),
                pd.concat([default_data_frame, pd.DataFrame(content, columns=["a", "b", "c"])]).reset_index(drop=True),
            )
```

Reply:
If the user tries to append to ParquetDN without installing fastparquet, Python will raise the error saying that parquet is not installed.

We will also have a warning in the doc saying that fastparquet is required for appending.